src/vtk-1-fixes.patch

```diff
@@ -0,0 +1,538 @@
+This file is part of MXE.
+See index.html for further information.
+
+Contains ad hoc patches for cross building.
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Muellni <mmuellen@gmx.de>
+Date: Tue, 25 Oct 2011 23:32:42 +0200
+Subject: [PATCH] add file TryRunResults.cmake
```

Comment:
I wanted to try replacing these using the prefixed cmake wrapper and the [common `TryRunResults.cmake`](https://github.com/mxe/mxe/blob/master/src/cmake/modules/TryRunResults.cmake)